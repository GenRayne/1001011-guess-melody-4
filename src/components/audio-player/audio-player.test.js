import React from 'react';
import renderer from 'react-test-renderer';
import AudioPlayer from './audio-player.jsx';

const src = `https://upload.wikimedia.org/wikipedia/commons/4/4e/BWV_543-fugue.ogg`;
const isPlaying = false;

describe(`render AudioPlayer`, () => {
  it(`renders AudioPlayer`, () => {
    const tree = renderer
      .create(
          <AudioPlayer src={src} isNowPlaying={isPlaying} />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
