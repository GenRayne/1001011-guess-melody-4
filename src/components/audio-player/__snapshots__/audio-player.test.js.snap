// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render AudioPlayer renders AudioPlayer 1`] = `
Array [
  <button
    aria-label="Play"
    className="track__button track__button--play"
    disabled={true}
    onClick={[Function]}
    type="button"
  />,
  <div
    className="track__status"
  >
    <audio />
  </div>,
]
`;
